tools/nnwrap/generate_wrappers.py

```diff
@@ -74,7 +74,7 @@
 def wrap_function(name, type, arguments):
     cname = 'THNN_' + type + name
     declaration = ''
-    declaration += 'extern "C" void ' + cname + \
+    declaration += 'TH_API void ' + cname + \
```

Comment:
it's not really TH_API. why do you do that?

Reply:
The export decorations for THNN is currently TH_API. If these docorations aren't used, the scripts in csrc will fail to build.